packages/react-components/src/organisms/EventMaterials.tsx

```diff
@@ -37,6 +38,14 @@ const EventMaterials: React.FC<EventMaterialsProps> = ({
     return null;
   }
 
+  if (
+    [notes, videoRecording, presentation, meetingMaterials].every(
```

Comment:
Maybe this should be moved somewhere else. I have to do the same check for toasts although i used filter instead of every. Whoops.

Reply:
somewhere else as in, `containsOnlyNulls`?

Reply:
I was thinking the array of "materials"